tests/Common/DataFixtures/Executor/ORMExecutorTest.php

```diff
@@ -52,8 +52,7 @@ public function testExecuteWithPurge(): void
         $em     = $this->getMockSqliteEntityManager();
         $purger = $this->getMockPurger();
         $purger->expects($this->once())
-            ->method('purge')
-            ->willReturn(null);
+            ->method('purge');
```

Comment:
Updated because `purge()` return type is `void`.

Reply:
We should backport that change to 1.8.x.